docs/av-stack-tidy.1.md

```diff
@@ -0,0 +1,17 @@
+# av-stack-tidy 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-stack-tidy - Record changes to the repository with commits
```

Comment:
I'm not sure I understand this statement in context of this command.

If this is a short description, wouldn't this be more clear:
```
Clean up deleted branches from the stack
```